xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeMovieAssets.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "DirectoryNodeMovieAssets.h"
+
+#include "FileItem.h"
+#include "FileItemList.h"
+#include "QueryParams.h"
+#include "guilib/LocalizeStrings.h"
+#include "utils/URIUtils.h"
+#include "video/VideoDatabase.h"
+#include "video/VideoManagerTypes.h"
+
+using namespace XFILE::VIDEODATABASEDIRECTORY;
+
+CDirectoryNodeMovieAssets::CDirectoryNodeMovieAssets(const std::string& strName,
+                                                     CDirectoryNode* pParent)
+  : CDirectoryNode(NodeType::MOVIE_ASSETS, strName, pParent)
+{
+}
+
+bool CDirectoryNodeMovieAssets::GetContent(CFileItemList& items) const
+{
+  CVideoDatabase videodatabase;
+  if (!videodatabase.Open())
+    return false;
+
+  CQueryParams params;
+  CollectQueryParams(params);
+
+  const int details{items.HasProperty("set_videodb_details")
+                        ? items.GetProperty("set_videodb_details").asInteger32()
+                        : VideoDbDetailsNone};
+
+  const std::string path{BuildPath()};
+
+  bool bSuccess = videodatabase.GetMoviesNav(
+      path, items, params.GetGenreId(), params.GetYear(), params.GetActorId(),
+      params.GetDirectorId(), params.GetStudioId(), params.GetCountryId(), params.GetSetId(),
+      params.GetTagId(), SortDescription(), details);
+
+  videodatabase.Close();
+
+  if (params.GetVideoAssetType() == -2 && items.Size() && items[0]->HasVideoExtras() &&
```

Comment:
Could we introduce a constant with a speaking name to get rid of the magic number `-2`?